Kirstie-Whitaker/abstracts_bios_2019.md

```diff
@@ -252,34 +256,31 @@ She is the chair of the Turing Institute's Ethics Advisory Group, a Fulbright sc
 Kirstie was named, with her collaborator Petra Vertes, as a [2016 Global Thinker](https://gt.foreignpolicy.com/2016/profile/petra-vertes-and-kirstie-whitaker) by Foreign Policy magazine.
 You can find more information at her lab website: [whitakerlab.github.io](https://whitakerlab.github.io/).
 
-**Slides**: *Figshare link to slides (if applicable)*
+**Slides**: https://doi.org/10.5281/zenodo.3238189
 
 
-### Autistica Discover Conference
+### Talk in OHBM Open Science symposium
```

Comment:
Hi,Kirstie.It seems like you deleted somthing here. :sob:

Reply:
Thanks @minnieho1115! Restoring now!